src/services/rest/http-client.js

```diff
@@ -3,6 +3,8 @@ import url from 'url';
 // see getDefaultHTTPModule for additional imports due to webpack
 
 const IS_NODE = typeof window === 'undefined';
+const HTTP_200_OK_RANGE_START = 200;
```

Comment:
I don't agree we should fail anything in the 200 range.
"2xx Success
This class of status codes indicates the action requested by the client was received, understood, accepted, and processed successfully"